hnn_core/gui/gui.py

```diff
@@ -1493,10 +1498,8 @@ def get_cell_param_default_value(cell_type_key, param_dict):
 
 def add_drive_tab(params, log_out, drives_out, drive_widgets, drive_boxes,
                   tstop, layout):
-    net = jones_2009_model(params)
-
-    drive_specs = _extract_drive_specs_from_hnn_params(
```

Comment:
We can likely deprecate (or just remove since it's private) this function in the future

Reply:
Sure. Just need to check if it's only use is in the GUI.